_stories/atoms/RadioButton/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { optionalSelect } from '../../../components/utils/optionalSelect';
+
+import readme from './README.md';
+import RadioButton from '../../../components/atoms/RadioButton';
+
+const sizeOptions = {
+    xs: 'xs',
+    sm: 'sm',
+    'No Value': ''
+};
+
+const contextOptions = {
+    success: 'success',
+    warning: 'warning',
+    danger: 'danger',
+    'No Value': ''
+};
+
+const component = () => (
+    <RadioButton
+        size={optionalSelect('Size', sizeOptions, '')}
+        className={text('Class Name', 'custom-class')}
+        label={text('Label', 'Yes')}
+        value={text('Value', 'yes')}
+        onChange={action('change')}
+        onBlur={action('blur')}
+        context={select('Context', contextOptions, 'default')}
```

Comment:
i think you want to use optionalSelect here instead of select

Reply:
and this and RadioGroup should use'' instead of 'default'. 'default' is causing it to show the first option in the story

Reply:
done